menu.rb

```diff
@@ -0,0 +1,28 @@
+# creates a blank line
+puts " "
+
+# prints to separate lines
+puts "Welcome to Chance's Diner!"
+puts "Tonight's specials are: "
+
+# creates dash lined separation between welcome message and menu
+puts "--" * 15
+
+# arrays from which menu items are pulled
+descriptive = ["Buttery", "Watery", "Spicy", "Crispy", "Moist", "Chunky", "Crusty", "Mushy", "Soggy", "Tongue-scorching", "Mind-blowing"]
+style = ["Brazilian", "Chinese", "Mexican", "Cajun", "Bacon-wrapped", "Cream Filled", "Deep Fried", "Cheese Crusted", "Sausage Stuffed", "Five Spice", "Beer Mop"]
+main = ["Lamb Burger", "Chicken Feet", "Veal Steak", "Catfish", "Ravioli", "Octopus", "Sea Urchin", "Chicken Skin", "Gizzard", "Geoduck", "Beef Tongue"]
```

Comment:
Nice variable names! You might want to be even more descriptive with `main`, maybe `main_dish`?